mro/__init__.py

```diff
@@ -140,11 +110,17 @@ def none_transform(column_default, data_type):
         cursor2.execute("select * from information_schema.columns where table_name='" + table_name + "';")
         connection.commit()
 
-        col_data = []
-
+        columns = []
         for column in cursor2:
+            col_data = {}
             column_name = column[3]
-            data_type = type_map[column[7]]
+            postgres_type = column[7]
+            if postgres_type == 'USER-DEFINED':
+                postgres_type = column[27]
```

Comment:
I know you're just following the existing pattern, but any chance of using some named constants here instead of code like `column[27]'?